ListOfBooks/BooksViewController/Models/BookCategory.swift

```diff
@@ -1,15 +1,18 @@
 import Foundation
 
-struct BookResult: Decodable {
-    let results: [BookCategory]
-}
-
 struct BookCategory: Decodable {
     let listName: String
     let oldestPublishedDate: String
+    let dateFormatterPrint: Date?
```

Comment:
Try this peace of code in your `BookCategory` and call it on UI
`    func displayOldestPublishedDate() -> String {
        let dateFormatter = DateFormatter()
        dateFormatter.dateFormat = "yyyy-mm-dd"
        guard let date = dateFormatter.date(from: oldestPublishedDate) else {
            return "N/A"
        }
        dateFormatter.dateFormat = "MMM d, yyyy"
        return dateFormatter.string(from: date)
    }`